__tests__/fluentBit.spec.ts

```diff
@@ -1,42 +1,23 @@
+import { readFileSync } from 'fs';
 import { FluentBitSchema } from '../index';
+import { TokenError } from '../src/TokenError';
 import { cases } from '../__fixtures__/fluentBitCases';
 
 jest.mock('uuid', () => ({ v4: () => 'UNIQUE' }));
 
 describe('fluentBit', () => {
   it('Fails if config is empty', () => {
-    expect(() => new FluentBitSchema('       ')).toThrowErrorMatchingInlineSnapshot('"Invalid config file"');
-  });
-
-  it('Fails if config has no fields', () => {
-    expect(() => new FluentBitSchema('# some comment')).toThrowErrorMatchingInlineSnapshot(
-      '"This file is not a valid Fluent Bit config file"'
+    expect(() => new FluentBitSchema('       ', '/file/path.conf')).toThrowErrorMatchingInlineSnapshot(
```

Comment:
We probably should test a completely empty (zero length) file as well. It generally finds the issues with assuming >0 length - it found a few issues in the YAML fuzz testing for Fluent Bit for example.

Reply:
We should also add a test for garbage input, i.e. invalid encoding and just general nonsense to make sure we pick this up.

Reply:
good test cases as well.

Reply:
https://github.com/calyptia/fluent-bit-config-parser/issues/19